pkg/reconciler/managed/reconciler.go

```diff
@@ -311,8 +311,8 @@ type ExternalObservation struct {
 	// ResourceLateInitialized should be true if the managed resource's spec was
 	// updated during its observation. A Crossplane provider may update a
 	// managed resource's spec fields after it is created or updated, as long as
-	// the updates are limited to setting previously unset fields, and adding
-	// keys to maps. Crossplane uses this information to determine whether
+	// the updates are limited to setting previously unset fields, adding
+	// keys to maps and adding data to slices. Crossplane uses this information to determine whether
```

Comment:
Nit: Could you please wrap this comment at 80 cols.

Reply:
@negz done